CHANGELOG.md

```diff
@@ -27,6 +28,150 @@
 - [v0.1.0-rc2](#v010-rc2)
 - [v0.1.0-rc1](#v010-rc1)
 
+# v0.8.0-rc1
+
+The working group expects that this release candidate is quite close to the final
+v0.8.0 release. However, breaking API changes are still possible.
+
+This release candidate is suitable for implementors, but the working group does
+not recommend shipping products based on a release candidate API due to the
+possibility of incompatible changes prior to the final release.
+
+## Major Themes
+
+### GAMMA (Service Mesh)
+The GAMMA work for supporting service mesh use cases is _experimental_ in
+`v0.8.0`. It is possible that it will change; we do not recommend it in
+production at this point.
+
+Since there will usually only be one mesh active in the cluster, the Gateway and
+GatewayClass resources are not used for Service Mesh; instead, individual route
+resources (such as HTTPRoute) are associated directly with Service resources,
+permitting the mesh to manage traffic from any traffic directed to that Service
+while preserving the role-oriented nature of the Gateway API.
+
+In Gateway API v0.8.0, GAMMA support for service mesh is **experimental**. We
+encourage working with it and providing feedback, but you **must** be prepared
+for changes in the GAMMA APIs.
```

Comment:
@keithmattix @kflynn @howardjohn could use some help on this section if any of you have ideas.